shlr/spp/r_api.h

```diff
@@ -1,22 +1,12 @@
 #ifndef S_STRBUF_H
 #define S_STRBUF_H
 
-#define R_FREE(x) { free(x); x = NULL; }
```

Comment:
??? no. spp should not be touched, it's part of another project, see radare/spp.git

Reply:
R_FREE is already defined so is r2 util api needs to be transfered there ?

Reply:
there are 2 problems:
1) any change in the shlr/spp directory should first land in radare/spp git project and only then be synced in r2 codebase
2) spp should compile as a standalone binary and it won't compile if you remove these things.

The ideal solution would be to have r_util as a separate project that can be imported and used by both radare2 and spp.

Reply:
For sure that s the ideal but dunno what @radare would think of it ;-)

Reply:
I know, but still, you shouldn't touch these lines in shlr/spp.